ui/components/capture/src/main/java/com/google/jetpackcamera/ui/components/capture/debug/DebugOverlayComponents.kt

```diff
@@ -59,32 +62,134 @@ import com.google.jetpackcamera.ui.components.capture.DEBUG_OVERLAY_SET_ZOOM_RAT
 import com.google.jetpackcamera.ui.components.capture.DEBUG_OVERLAY_SET_ZOOM_RATIO_TEXT_FIELD
 import com.google.jetpackcamera.ui.components.capture.DEBUG_OVERLAY_SHOW_CAMERA_PROPERTIES_BUTTON
 import com.google.jetpackcamera.ui.components.capture.DEBUG_OVERLAY_VIDEO_RESOLUTION_TAG
+import com.google.jetpackcamera.ui.components.capture.LOGICAL_CAMERA_ID_TAG
+import com.google.jetpackcamera.ui.components.capture.PHYSICAL_CAMERA_ID_TAG
+import com.google.jetpackcamera.ui.components.capture.R
+import com.google.jetpackcamera.ui.components.capture.ZOOM_RATIO_TAG
 import com.google.jetpackcamera.ui.uistate.capture.DebugUiState
 import kotlin.math.abs
 
 private const val TAG = "DebugOverlayComponents"
 
 @Composable
-fun DebugOverlayToggleButton(modifier: Modifier = Modifier, toggleIsOpen: () -> Unit) {
-    TextButton(modifier = modifier.testTag(DEBUG_OVERLAY_BUTTON), onClick = { toggleIsOpen() }) {
+fun DebugDialogContainerToggle(toggleIsOpen: () -> Unit, modifier: Modifier = Modifier) {
+    Button(modifier = modifier.testTag(DEBUG_OVERLAY_BUTTON), onClick = { toggleIsOpen() }) {
         Text(text = "Debug")
     }
 }
 
 @Composable
-fun DebugOverlayComponent(
+private fun CurrentCameraIdText(physicalCameraId: String?, logicalCameraId: String?) {
+    Column(horizontalAlignment = Alignment.CenterHorizontally) {
+        Row {
+            Text(text = stringResource(R.string.debug_text_logical_camera_id_prefix))
+            Text(
+                modifier = Modifier.testTag(LOGICAL_CAMERA_ID_TAG),
+                text = logicalCameraId ?: "---"
+            )
+        }
+        Row {
+            Text(text = stringResource(R.string.debug_text_physical_camera_id_prefix))
+            Text(
+                modifier = Modifier.testTag(PHYSICAL_CAMERA_ID_TAG),
+                text = physicalCameraId ?: "---"
+            )
+        }
+    }
+}
+
+@Composable
+private fun ZoomRatioText(modifier: Modifier = Modifier, primaryZoomRatio: Float?) {
+    Text(
+        modifier = modifier
+            .testTag(ZOOM_RATIO_TAG),
+        text = stringResource(id = R.string.zoom_ratio_text, primaryZoomRatio ?: 1f)
+    )
+}
+
+@Composable
+fun DebugOverlay(
+    modifier: Modifier = Modifier,
+    onChangeZoomRatio: (Float) -> Unit,
+    onSetTestPattern: (TestPattern) -> Unit,
+    toggleIsOpen: () -> Unit,
+    debugUiState: DebugUiState.Enabled,
+    vararg extraControls: @Composable () -> Unit
+) {
+    Box(modifier = modifier.fillMaxSize()) {
+        DebugConsole(
```

Comment:
This seems to overlap the indicators. Can it be moved down so it doesn't overlap those indicators anymore? Also, maybe it makes sense to have a transparent background for these so that when the text is on top of a very light background showing in the viewfinder, it's still legible.